src/blog/2025-06-01-oxlint-stable.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Oxlint v1.0
+outline: deep
+authors:
+  - boshen
+  - cameron
+---
+
+<AppBlogPostHeader />
+
+Oxlint is finally stable since its first announcement back in December 2023.
+
+Oxlint is a Rust-powered linter for JavaScript and TypeScript that ships with 500+ built-in rules and is designed to be fast and simple to adopt.
+
+We now have a dedicated full-time maintainer ([@cameron](https://github.com/camc314)) and a growing core team working on maintaining and improving the linter.
+
+## Real-World Impact
+
+We are extremely proud of the performance of Oxlint and its impact on real large codebases, which has led to reduced CI costs.
+
+We are thankful for the [5.2k early adopters](https://github.com/oxc-project/oxc/network/dependents), and also companies such as:
+
+- _Shopify_'s Front-End Platform team uses oxlint in the Shopify Admin.
+- _Airbnb_ uses to enforce rules such as `oxc/no-barrel-file` and `import/no-cycle` on their 11 million lines of code. ESLint's implementation of these rules times out.
+- _Mercedes-Benz_ observed a 71% decrease in lint time when swapping ESLint to oxlint, with some project seeing up to a 97% speed up.
+
```

Comment:
We could also mention open source projects using Oxlint successfully, e.g., bun, preact, etc.
IMO that is highlight-worthy!